site/en/Test.md

```diff
@@ -0,0 +1,41 @@
+
+The recognition quality was only checked on individual pages by the respective pilot libraries, as there is no ground truth available for the test books. Overall, the results of this first test run are promising. The Mannheim University Library has tested OCR-D with a focus on tesseract processors on five prints from the 16th to 19th century. On antiqua prints from the 17th and 18th centuries and a blackletter text from the 19th century, as expected, the best results of - in the case of the antiqua - significantly less than 0.1 CER were achieved for the raw data, whereas the blackletter prints from the 17th century was slightly above 0.1 CER. The greatest challenge was the 16th century bleckletter, where only a CER of just under 0.16 was reached. The BBAW provides a comprehensive insight into their testing, by making their [report and data publicly available](https://github.com/tboenig/ocrd_bbaw_pilotbibliothek).
+
+The OCR-D testers formulate some desiderata especially with regard to documentation, quality and usability of the processors as well as their future scalability. The requirements for the documentation of the OCR-D software have already been implemented to a large extent, though documentation as a whole is regarded as a continuous task which must successively include above all practical experience in the application of the OCR-D software. For the processors, some improvements would still be desirable, especially in layout recognition and post-correction. The corresponding developments of the OCR-D module projects could only be tested to a limited extent due to their development stage or their special technical requirements (GPU). Hopefully, the above mentioned desiderata can be met with their results or with further models. For the use of the OCR-D software in mass digitisation, the runtime of several processors - as originally planned for the third project phase - still needs to be optimised, and the possibilities for parallelisation should also be expanded. 
+
+The testers positively emphasized the modular and transparent structure of the OCR-D software, which distinguishes it in particular from other OCR solutions and allows the its users to confiure optimal workflows for particular use cases. Furthermore, the open source available Python programs can be further developed by experts specialized in their respective fields and can be used for experiments on the OCR workflow without extensive programming work. In case of questions and problems, the developers quickly provide low-threshold support. All in all, it is comparatively easy to initiate the robustly running OCR-D full-text generation process, which is still in need of further optimization but already delivers promising results.
```

Comment:
the open source available Python programs -> all OCR-D source code is freely available and can be adapted in-house to be used for experiments ...